feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/viewmodel/SessionsViewModel.kt

```diff
@@ -63,6 +64,9 @@ class SessionsViewModel @Inject constructor(
 
     private val filterLiveData: MutableLiveData<Filters> = MutableLiveData(Filters())
 
+    // map of the given Day and the current ongoing sessions index in the list of that day
+    val currentSessionMap = MutableLiveData<Map<SessionPage.Day, Int>>()
```

Comment:
Thank you for PR 📜 
Due to the complexity of the data flow, I don't want to publish LiveData for Fragment other than UiModel.

Reply:
Let me think a bit about what to do.
If you have any ideas, please try to implement them.